docs/tutorials/tutorial-hdfs-logs-distributed-search-aws-s3.md

```diff
@@ -95,6 +95,14 @@ default_index_root_uri: ${S3_PATH}
 " > config.yaml
 ```
 
+> You can also pass environment variables directly:
```

Comment:
I think this tutorial deserves some rewriting. I'll take care of it before the next release. You can dismiss those changes for now.